Tag log messages

deldesir opened the pull request:
Capture both standard output (stdout) and standard error (stderr) separately. The script then logs the stdout lines with the "Info" level and stderr lines with the "Error" level.

holta commented:
@deldesir do you offer CSS lessons?!

Where do I and @thadk sign up 😄

deldesir commented:
Just learning by doing...

holta commented:
@deldesir should this PR be labeled as a [DRAFT](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests) (if indeed it's an experimental work-in-progress?)

deldesir committed:
Merge branch 'iiab:master' into deldesir-logging

deldesir committed:
Remove polling file

deldesir committed:
Tag log msgs

The log messages are now tagged with "Info" or "Error" based on the nature of the message.

deldesir committed:
Update lb-wrapper

Capture both standard output (stdout) and standard error (stderr) separately. The script then logs the stdout lines with the "Info" level and stderr lines with the "Error" level.
